usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/MealManagerClientTest.java

```diff
@@ -0,0 +1,145 @@
+        mealList.add(meal);
+        jsonMealData = new StringBuilder("{\n"
+            + "      \"mealName\": \"Ultra Asparagus\",\n"
+            + "      \"kcal\": 54.\n"
+            + "    }\n");
+        jsonAllMeals = new StringBuilder("[\n"
+            + "{\n"
+            + "    \"date\": \"2020-02-13T10:30:00\",\n"
+            + "    \"body\": {\n"
+            + "      \"mealName\": \"Ultra Asparagus\",\n"
+            + "      \"kcal\": 54\n"
+            + "    }\n"
+            + "  }"
+            + "   \n"
+            + "]");
+    }
+
+    @Test
+    public void createMeal() throws ExecutionException, InterruptedException {
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(STATUS_OK);
+
+        int responseCode = client.createMeal(ACCESS_TOKEN, owner, petName, meal);
+        assertEquals("Should return response code 200", STATUS_OK_INT, responseCode);
+    }
+
+    @Test
+    public void deleteByDate() throws ExecutionException, InterruptedException {
+
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(STATUS_OK);
+
+        int responseCode = client.deleteByDate(ACCESS_TOKEN, owner, petName, date);
+        assertEquals("Should return response code 200", STATUS_OK_INT, responseCode);
+    }
+
+    @Test
+    public void deleteAllMeals() throws ExecutionException, InterruptedException {
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(STATUS_OK);
+
+        int responseCode = client.deleteAllMeals(ACCESS_TOKEN, owner, petName);
+        assertEquals("Should return response code 200", STATUS_OK_INT, responseCode);
+    }
+
+    @Test
+    public void getMealData() throws ExecutionException, InterruptedException {
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(jsonMealData);
+
+        MealData response = client.getMealData(ACCESS_TOKEN, owner, petName, date);
+        assertEquals("Should return meal data", response, mealData);
+    }
+
+    @Test
+    public void getAllMealData() throws ExecutionException, InterruptedException {
+        given(taskManager.resetTaskManager()).willReturn(taskManager);
+        given(taskManager.execute(anyString(), anyString())).willReturn(taskManager);
+        given(taskManager.get()).willReturn(jsonAllMeals);
+
+        List<Meal> response = client.getAllMealData(ACCESS_TOKEN, owner, petName);
+        assertEquals("Should return a meal data list", response, mealList);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Should return a meal data list" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5327591)